### PR TITLE
PS-4579 Documentation on Generic tarball is wrong

### PR DESCRIPTION
**Pull request by patrickbirch**
in the installation doc, added a note that demonstrates how to find the ssl files based on OS

### CODE REVIEW

File: doc/source/installation.rst
Comment (tplavcic) [inline doc/source/installation.rst]: I would better replace this with: `locate libssl | grep "^/usr/lib/"`
since the command you listed lists the libssl libraries from the installed snaps also, which probably should not be used for this.
Here's how it looks on ubuntu bionic:
```
$ locate libssl | grep /usr/lib/
/snap/core18/1650/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
/snap/core18/1650/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/snap/core18/1668/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
/snap/core18/1668/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/snap/gnome-3-28-1804/110/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/snap/gnome-3-28-1804/116/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/usr/lib/firefox/libssl3.so
/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/usr/lib/x86_64-linux-gnu/libssl3.so
```
this one gives shorter list without the ones from the snaps (might be less confusing for users).
```
$ locate libssl | grep "^/usr/lib/"
/usr/lib/firefox/libssl3.so
/usr/lib/x86_64-linux-gnu/libssl.so.1.0.0
/usr/lib/x86_64-linux-gnu/libssl.so.1.1
/usr/lib/x86_64-linux-gnu/libssl3.so
```

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: doc/source/installation.rst
Comment (patrickbirch) [reply doc/source/installation.rst]: Done. I changed the command to match your suggestion.

**Review by tplavcic — APPROVED** [review]
LGTM

**Comment by patrickbirch** [comment]
waiting for approval or comments

File: doc/source/installation.rst
Comment (hrvojem) [inline doc/source/installation.rst:67]: Debian Wheezy has been deprecated for a while now, please use supported distro for examples

Comment (patrickbirch) [reply doc/source/installation.rst:67]: Updated in the 5.7 version

**Pull request merged**
